model_hub/examples/huggingface/token-classification/ner_utils.py

```diff
@@ -0,0 +1,128 @@
+"""
+These functions extract parts of the code from the named-entity recognition example
+in huggingface transformers.  The source example code changes rapidly due to frequent refactorings
+but the main structure should be similar.
+See the original example at:
+https://github.com/huggingface/transformers/blob/master/examples/token-classification/run_ner.py
+
+Note this code is separated from ner_trial.py since it is not determined specific code and
+is necessary even if implementing outside of determined.
+"""
+from typing import Any, Dict, List
+
+import datasets
+import numpy as np
+from attrdict import AttrDict
```

Comment:
(throughout) Let's stick to the google coding standard for imports, which is to always import the module rather than symbols in the module (with the special exception of the `typing` module, for some reason).

We try to enforce this on all new code in Determined, though we've always been real lax with it in examples.